android/app/src/main/java/org/uwaterloo/subletr/pages/managelisting/ManageListingPageView.kt

```diff
@@ -0,0 +1,806 @@
+						triggerOnValueChange = { viewModel.attemptUpdateStream.onNext(false) },
+						changeValue = {
+							viewModel.editableFieldsStream.onNext(uiState.editableFields.copy(bathroomsAvailable = it))
+						},
+						attemptCreate = uiState.attemptUpdate,
+						prefix = null,
+					)
+					NumericalInputTextField(
+						modifier = Modifier.weight(1f),
+						labelId = R.string.bathrooms_in_unit,
+						uiStateValue = uiState.editableFields.bathroomsTotal ?: 0,
+						triggerOnValueChange = { viewModel.attemptUpdateStream.onNext(false) },
+						changeValue = {
+							viewModel.editableFieldsStream.onNext(uiState.editableFields.copy(bathroomsTotal = it))
+						},
+						attemptCreate = uiState.attemptUpdate,
+						prefix = null,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.s)),
+				)
+
+				RoundedExposedDropdown(
+					dropdownItems = EnsuiteBathroomOption.values(),
+					labelId = R.string.ensuite_bathroom,
+					selectedDropdownItem = if (uiState.editableFields.bathroomsEnsuite == 1)
+						EnsuiteBathroomOption.YES
+					else EnsuiteBathroomOption.NO,
+					dropdownItemToString = { stringResource(id = it.stringId) },
+					setSelectedDropdownItem = {
+						viewModel.editableFieldsStream.onNext(uiState.editableFields.copy(bathroomsEnsuite =
+						if (it == EnsuiteBathroomOption.YES) 1 else 0))
+					},
+				)
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.m)),
+				)
+
+				Row(
+					modifier = Modifier.fillMaxWidth(),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(id = R.string.additional_information),
+						color = MaterialTheme.subletrPalette.primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.xs)),
+				)
+
+				RoundedExposedDropdown(
+					dropdownItems = ListingForGenderOption.values(),
+					labelId = R.string.gender,
+					selectedDropdownItem = uiState.editableFields.gender?.getGender() ?: ListingForGenderOption.ANY,
+					dropdownItemToString = { stringResource(id = it.stringId) },
+					setSelectedDropdownItem = {
+						viewModel.editableFieldsStream.onNext(
+							uiState.editableFields.copy(gender = it.getKey())
+						)
+					},
+				)
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.s)),
+				)
+
+				RoundedExposedDropdown(
+					dropdownItems = HousingType.values(),
+					labelId = R.string.housing_type,
+					selectedDropdownItem = uiState.editableFields.residenceType?.toHousingType() ?: HousingType.OTHER,
+					dropdownItemToString = { stringResource(id = it.stringId) },
+					setSelectedDropdownItem = {
+						viewModel.editableFieldsStream.onNext(
+							uiState.editableFields.copy(residenceType = it.toResidenceType())
```

Comment:
LGTM! Just one small question, are we going to use the ResidenceType instead of Housing Type? In this case, I need to fix the housing type filter.

Reply:
so the backend always takes residence type whereas we are using housing type which have different values. So you have to send to the backend a residence type, ideally they would be the same, or at least have the same values but because housingtype and more we were using it on the front end

its just a small backend change I think to add more types to residence type